Tests/govuk_ios/govuk_ios_unit_tests/Specs/ViewModels/RecentActivitiesViewModelTests.swift

```diff
@@ -123,62 +125,222 @@ struct RecentActivitiesViewModelTests {
         activitiesArray.append(activityTwo)
         activitiesArray.append(activityThree)
 
-        let structure = sut.sortActivites(activities: activitiesArray)
+        sut.sortActivites(activities: activitiesArray)
+
+        #expect(sut.model.todaysActivites.count == 0)
+        #expect(sut.model.recentMonthActivities.count == 3)
+        #expect(sut.model.currentMonthActivities.count == 0)
+    }
+
+    @Test
+    func deleteActivities_deletesAllActivities() async throws {
+        let coreData = CoreDataRepository.arrange(
+            notificationCenter: .default
+        ).load()
+
+        let activityOne = ActivityItem(context: coreData.backgroundContext)
```

Comment:
Looks like we have same local objects being created for different tests?
We could create these once outside the test function maybe to reduce line count?

Reply:
fixed